curriculum/challenges/english/10-coding-interview-prep/project-euler/problem-139-pythagorean-tiles.md

```diff
@@ -12,27 +12,31 @@ Let (a, b, c) represent the three sides of a right angle triangle with integral
 
 For example, (3, 4, 5) triangles can be placed together to form a 5 by 5 square with a 1 by 1 hole in the middle and it can be seen that the 5 by 5 square can be tiled with twenty-five 1 by 1 squares.
 
-However, if (5, 12, 13) triangles were used then the hole would measure 7 by 7 and these could not be used to tile the 13 by 13 square. Given that the perimeter of the right triangle is less than one-hundred million, how many Pythagorean triangles would allow such a tiling to take place?
+<img class="img-responsive center-block" alt="two 5 x 5 squares: one with four 3x4x5 triangles placed to create 1x1 hole in the middle; second with twenty-five 1x1 squares" src="https://cdn.freecodecamp.org/curriculum/project-euler/pythagorean-tiles.png" style="background-color: white; padding: 10px;">
+
+However, if (5, 12, 13) triangles were used, the hole would measure 7 by 7, these could not be used to tile the 13 by 13 square.
```

Comment:
```suggestion
However, if (5, 12, 13) triangles were used, the hole would measure 7 by 7, and these could not be used to tile the 13 by 13 square.
```

Reply:
@scissorsneedfoodtoo "and" were removed here per @Sembauke's request - https://github.com/freeCodeCamp/freeCodeCamp/pull/42731#discussion_r663831061. Maybe there's way to reword this and avoid place where "and" would both kind of fit and not fit.

Reply:
I tried making two different sentences but haven't found a great one.

Reply:
```suggestion
However, if (5, 12, 13) triangles were used, the hole would measure 7 by 7. This means that you can not tile the 13 by 13 square.
```

Reply:
Yeah, this would be a bit tricky to split into two sentences without some extra wording to preserve clarity.

My understanding is that, with (5, 12, 13) triangles, the 7 by 7 hole that they would produce could not be used to tile the 13 by 13 square.

@Sembauke's suggestion above could work, though there might be some confusion about why you can't tile the 13 by 13 square, or what you're trying to tile it with.

Something like this could work:

"However, if (5, 12, 13) triangles were used, the hole would measure 7 by 7. These 7 by 7 squares could not be used to tile the 13 by 13 square."

But it's so close to the original wording that I think it's better to stick with the original sentence, "However, if (5, 12, 13) triangles were used, the hole would measure 7 by 7, and these could not be used to tile the 13 by 13 square."

Open to any suggestions, though! There is probably a way to communicate this with two sentences that I'm just not seeing.

Reply:
To be honest, the explanation of the Euler problem 139 seems a little confusing to me overall, but that might be because of my own English skill. Although the Euler curriculum is for the intermediate coder. 

The first paragraph seems a little wordy to me too: 
For example, (3, 4, 5) triangles can `be` placed together `to` form a 5 by 5 square with `a`  1 by 1 hole in `the` middle and it can be seen `that the` 5 by 5 square can `be` tiled with twenty-five 1 by 1 squares.

This seems to be a really long correlating sentence that might confuse the nonnative speaker. I think we might need to change that too. 

There also seems to be no explanation for this part of the sentence `For example, (3, 4, 5) triangles`  does this mean the number of triangles?

@scissorsneedfoodtoo what is your opinion on this?

To come back to your suggestion of the current sentence we are debating on, I like how it sounds.

"However, if (5, 12, 13) triangles were used, the hole would measure 7 by 7. These 7 by 7 squares could not be used to tile the 13 by 13 square."

Reply:
(3, 4, 5) are sides of the triangle

Maybe something like this:
>For example, triangles with sides (3, 4, 5) can be placed together to form a 5 by 5 square with a 1 by 1 hole in the middle ...

>However for 13 by 13 square, triangles with sides (5, 12, 13) could not be used, as the hole would need to measure 7 by 7.

Reply:
I agree

Reply:
I'm fine with whatever you all decide here - you made some decent points. I can see how it could be a little confusing. That's probably the case with many of these problems, as they are trying to describe complex things. I don't really have any issues with the [original wording](https://projecteuler.net/problem=139). If it were me, that's what I would go with.

Reply:
So are we fine with how this is then or are we will trying to come up with something?

Reply:
Sorry for the late response.

I'm also more inclined to stick with the original wording whenever possible. If we need to change the wording for porting, or because we get some complaints about the description being difficult to understand, then that's totally. But I think the original description is fine in this case.

That said, @Sembauke's suggestion would work, too: "However, if (5, 12, 13) triangles were used, the hole would measure 7 by 7. These 7 by 7 squares could not be used to tile the 13 by 13 square."

Reply:
I would also point to this one
> "However, if (5, 12, 13) triangles were used, the hole would measure 7 by 7. These 7 by 7 squares could not be used to tile the 13 by 13 square."

It seems clear enough

Reply:
@gikf can you change it to:
"However, if (5, 12, 13) triangles were used, the hole would measure 7 by 7. These 7 by 7 squares could not be used to tile the 13 by 13 square."

Reply:
```suggestion
However, if (5, 12, 13) triangles were used, the hole would measure 7 by 7. These 7 by 7 squares could not be used to tile the 13 by 13 square.
```
it seems this is what we are agreeing on